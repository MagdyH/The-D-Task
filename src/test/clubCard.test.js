import React from 'react';
import renderer from 'react-test-renderer';
import ClubCard from '../components/clubCard';


describe('ClubCard component', () => {
    it('should render snapshot', () => {
      const component = renderer.create(<ClubCard />);
  
      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });

});