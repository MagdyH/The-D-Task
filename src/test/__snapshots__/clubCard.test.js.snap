// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ClubCard component should render snapshot 1`] = `
<div
  className="col-sm-9 my-4"
/>
`;
